source/configure/calls-kubernetes.md

```diff
@@ -0,0 +1,163 @@
+# Deploy Calls Kubernetes
+
+```{include} ../_static/badges/allplans-cloud-selfhosted.md
+```
+
+This guide provides detailed information for deploying Mattermost Calls on Kubernetes environments.
+
+## Overview
+
+Mattermost Calls has been designed to integrate well with Kubernetes to offer improved scalability and control over the deployment. For Kubernetes deployments, the RTCD service is strongly recommended and is the only officially supported approach.
+
+## Architecture
+
+![Calls deployed in a Kubernetes cluster](../images/calls-deployment-kubernetes.png)
+
+This diagram shows how the RTCD standalone service can be deployed in a Kubernetes cluster. In this architecture:
+
+1. Calls traffic is handled by dedicated RTCD pods
+2. RTCD services are exposed through load balancers
+3. Scaling is managed through Kubernetes deployment configurations
+4. Call recording and transcription is handled by the calls-offloader service (see [Calls Offloader Setup and Configuration](calls-offloader-setup.md))
+
+If Mattermost isn't already deployed in your Kubernetes cluster and you want to use this deployment type, visit the [Kubernetes operator guide](/install/mattermost-kubernetes-operator.md).
+
+## Helm Chart Deployment
+
+The recommended way to deploy Calls-related components in a Kubernetes environment is to use the officially provided Helm charts:
+
+### RTCD Helm Chart
+
+The RTCD Helm chart deploys the RTCD service needed for call media handling:
+
+```bash
+helm repo add mattermost https://helm.mattermost.com
+helm repo update
+
+helm install mattermost-rtcd mattermost/mattermost-rtcd \
+  --set ingress.enabled=true \
+  --set ingress.host=rtcd.example.com \
+  --set service.annotations."service\\.beta\\.kubernetes\\.io/aws-load-balancer-backend-protocol"=udp \
+  --set rtcd.ice.hostOverride=rtcd.example.com
+```
+
+For complete configuration options, see the [RTCD Helm chart documentation](https://github.com/mattermost/mattermost-helm/tree/master/charts/mattermost-rtcd).
+
+### Calls-Offloader Helm Chart
+
+If you need call recording and transcription capabilities, deploy the calls-offloader service:
+
+```bash
+helm install mattermost-calls-offloader mattermost/mattermost-calls-offloader \
+  --set ingress.enabled=true \
+  --set ingress.host=calls-offloader.example.com
+```
+
+For complete configuration options, see the [Calls-Offloader Helm chart documentation](https://github.com/mattermost/mattermost-helm/tree/master/charts/mattermost-calls-offloader).
+
+## Kubernetes-Specific Configuration
+
+### Network Configuration
+
+For Kubernetes deployments, you need to ensure:
+
+1. UDP traffic is properly routed to RTCD pods (for media)
+2. TCP traffic can reach both the Mattermost pods and RTCD pods
```

Comment:
This reads as slightly ambiguous or overly generic. There needs to be a clear connectivity path between Mattermost and RTCD on the API port (TCP 8045).